### PR TITLE
Timeline styling

### PR DESCRIPTION
**Pull request by IsmatMahmood1**
https://trello.com/c/lRK3slOS/34-create-the-styling-of-the-timeline-page

### CODE REVIEW

File: src/components/Timeline.tsx [hunk 1]
```diff
@@ -1,17 +1,34 @@
-import "../styles/Timeline.scss";
-import { timelineData } from "./timelineData";
+import { timelineData, timelineHeaderData } from "./timelineData";
+import "../styles/constants.scss";
+import "../styles/timeline.scss";
 import { Card } from "./Card";
 
 export function Timeline() {
+
```
Comment (oskarwilliams) [inline src/components/Timeline.tsx]: new line not needed :)

File: src/styles/Timeline.scss [hunk 2]
```diff
@@ -1,15 +1,95 @@
 @import './constants.scss';
 
-.information-card-holder{
-    width:70vw;
+
```
Comment (oskarwilliams) [inline src/styles/Timeline.scss]: blank new line :)

File: src/styles/Timeline.scss [hunk 3]
```diff
@@ -1,15 +1,95 @@
...
+    }
+    .card-body{
+        font-size: 14px;
+    }
 }
 
-@media (max-width: 430px) {
-    .information-card-holder{
+@media (max-width: 426px) {
+
```
Comment (oskarwilliams) [inline src/styles/Timeline.scss]: Blank new line

File: src/components/Timeline.tsx [hunk 4]
```diff
@@ -1,17 +1,34 @@
-import "../styles/Timeline.scss";
-import { timelineData } from "./timelineData";
+import { timelineData, timelineHeaderData } from "./timelineData";
+import "../styles/constants.scss";
+import "../styles/timeline.scss";
```
Comment (oskarwilliams) [inline src/components/Timeline.tsx]: this should be capital t for timeline

**Review by oskarwilliams — COMMENTED** [review]
Can I see a screenshot of what this looks like at a couple different widths, as it looks very odd to me
desktop
mobile
tablet

**Review by oskarwilliams — CHANGES_REQUESTED** [review]
Meant to make it a request changes sorry

File: src/components/Card.tsx [hunk 5]
```diff
@@ -6,7 +6,7 @@ interface CardProps {
     imageSrc: string;
     href: string;
     title: string;
-    body: string;
+    body: string | null | undefined;
```
Comment (oskarwilliams) [inline src/components/Card.tsx]: not sure this is needed

**Review by oskarwilliams — CHANGES_REQUESTED** [review]

**Comment by IsmatMahmood1** [comment]
New screen 
desktop2
mobile2
mobile2Smaller
tablet2
shoots

**Review by oskarwilliams — APPROVED** [review]

**Pull request merged**